### ISSUE TITLE
cabal2arch generates incomplete PKGBUILD for haskell-glut

### ISSUE THREAD

**Issue by peti** [archhaskell/cabal2arch#16]
haskell-glut depends on "freeglut", but cabal2arch doesn't know this fact, and consequently fails to generate a PKGBUILD file that builds successfully.

**Comment by magthe** [comment]
This is what I get:

```
% cabal2arch http://hackage.haskell.org/packages/archive/GLUT/2.2.2.0/GLUT.cabal
% grep freeglut haskell-glut/PKGBUILD
depends=('ghc' 'haskell-opengl>=2.3' 'haskell-statevar' 'haskell-tensor' 'haskell-array' 'haskell-containers' 'freeglut')
```

This is with the following version of `cabal2arch`:

```
% pacman -Q cabal2arch 
cabal2arch 0.7.6-5
```

I suggest this issue can be closed.

**Comment by peti** [comment]
Unless I'm mistaken, Haskell Platform says that we ought to use GLUT 2.1.2.1. In your test, you have used version 2.2.2.0.

**Comment by magthe** [comment]
You are correct, but then this makes it a duplicate of issue #14 since it's yet another missing dependency in the CABAL file.

**Comment by peti** [comment]
I disagree that it is accurate to call this issue a "duplicate" of issue 14. It is true that the problems in haskell-opengl and haskell-glut are both caused by the same underlying problem, i.e. the cabal file doesn't specifiy a dependency that we need to specify in ArchLinux. However, that doesn't imply that fixing the missing dependency in haskell-opengl would automagically fix the missing dependency in haskell-glut, too! Both issues need to be dealt with, and I think it's unwise to close this bug -- as you did -- until that issue is actually resolved.

**Comment by magthe** [comment]
Good that you re-opened it.  Of course you are correct, this is a separate instance of the same underlying issue.  I created #22 to deal with adding the feature that would allow us to close this issue, and #14.

**Comment by magthe** [comment]
Depends on #22.
